src/runtime/contrib/tensorrt/tensorrt_builder.h

```diff
@@ -153,6 +153,9 @@ class TensorRTBuilder {
   /*! \brief Whether to automatically convert model to 16-bit floating point precision. */
   bool use_fp16_;
 
+  /*! \brief whether to automatically convert model to int8 precision */
```

Comment:
IIUC, `use_int8_` is exclusive with `use_fp16_`? If so, we should combine them to be a single variable like `target_dtype`.